### ISSUE TITLE
AL File Wizards for Permission Set, Permission Set Extension

### ISSUE THREAD

**Issue by rvanbekkum** [anzwdev/al-code-outline#265]
It would be nice to have AL File Wizards added for Permission Set and Permission Set Extension AL objects.

In this wizard, allow the user to enter:
* ID
* Name (i.e., Role ID, currently max. 20 chars)
* Caption (i.e., Role Name, max. 30 chars)
* `IncludePermissionSets` property
* Objects and the permissions to apply (R/I/M/D/X/...)
    * If an object is selected, set defaults appropriate for the object type which the user can edit

**Comment by anzwdev** [comment]
Hi Rob

Thank you for the suggestion. I've released a new version of the extension with PermissionSet and PermissionSetExtension wizards. I haven't implemented objects selector, I need to think about UI first. Currently wizard can only add all objects from the current extension, it is also possible to select other permission sets to include in the permission set.

I've also added a few code actions to use in permission sets and permission set extensions:
- sort permissions - it is available when cursor is on the first line of "Permissions" property
- sort permission sets - it is available when cursor is on the first line of "IncludedPermissionSets" property
- add all objects from current extension to permissions - it is available when cursor is on the first line of "Permissions" property and adds all objects from the current extension to the list of permissions. It works not only with permission sets or permission set extensions, but also with other object types.
